installer_scripts/install-yb-voyager

```diff
@@ -555,6 +560,10 @@ get_passed_options() {
 				ONLY_PG="true"; 
 				shift 
 				;;
+			-n | --no-debezium )
```

Comment:
This is a subjective thing but how about "skip-debezium" or "exclude-debezium" instead? "no-" prefix feels slightly weird imo in the context of installation.   
Also `-n` is a bit confusing. (Same with `-v` that we currently have as well... ). What if we want to add "no-ora2pg" or "skip-ora2pg" in the future?
Can you explore if we can have two-letter short-forms?

Reply:
`skip-debezium` sounds good. let me explore `two-letter short-forms`

Reply:
Doesn't look like that is possible.
```
For example, -ab is interpreted as two short options: -a and -b.
```

But i think what we can do is just have long forms for some of the options.

Reply:
I think on mac, long-form options don't work. Let's just choose some letter then in that case. maybe "-d"  for skip-debeizum?